test/e2e/test/elasticsearch/builder.go

```diff
@@ -448,6 +454,11 @@ func (b Builder) WithPodLabel(key, value string) Builder {
 	return b
 }
 
+func (b Builder) WithToleratedHealthCheckErrors(msgs ...string) Builder {
+	b.toleratedHealthCheckErrors = msgs
```

Comment:
Should we append instead? If one chains those only the last will be in effect which might be surprising.